CHANGELOG.rdoc

```diff
@@ -13,6 +13,7 @@ Compatibility:
 * Converting to multipart moves Content-* headers to the new part. (kirikak2)
 * Multipart Content-Type no longer includes a needless charset param. (kirikak2)
 * Replies prefix subject with "Re: " instead of "RE: " per 5322 3.6.5. (mashedcode)
+* Fix spec text and improve smtp_spec assertion (adbatista)
```

Comment:
No need for a changelog update since this is an internal improvement, not user-facing.

Reply:
ok, I'll remove :)